spec/example_app/app/controllers/docs_controller.rb

```diff
@@ -9,11 +16,23 @@ def index
   end
 
   def show
-    render_page "docs/#{params[:page]}"
+    render_correct_page
+  end
+
+  def find_special_file
+    SPECIAL_FILES.select { |page| page[:page] == params[:page] }.first
   end
 
   private
 
+  def render_correct_page
+    if !find_special_file.nil?
```

Comment:
Instead of the double negative, we could go:

```suggestion
    if find_special_file
```

as if there is a special file, we'll render it and if it's `nil` we'll drop into the `else`.

Reply:
Oh obviously, thanks!